src/data/moviesSlice.js

```diff
@@ -0,0 +1,27 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit"
+
+export const fetchMovies = createAsyncThunk('fetch-movies', async (apiUrl) => {
+    const response = await fetch(apiUrl)
+    return response.json()
+})
+
+const moviesSlice = createSlice({
+    name: 'movies',
+    initialState: { 
+        movies: [],
+        fetchStatus: '',
+    },
+    reducers: {},
+    extraReducers: (builder) => {
+        builder.addCase(fetchMovies.fulfilled, (state, action) => {
+            state.movies = action.payload
```

Comment:
if we will use `state.movies = action.payload?.results `, it will actually save only movies